indico/migrations/versions/20210916_1127_b36825c7869e_add_registration_tags.py

```diff
@@ -0,0 +1,44 @@
+"""Add registration tags
+
+Revision ID: b36825c7869e
+Revises: 9d00917b2fa8
+Create Date: 2021-09-16 11:27:39.335488
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'b36825c7869e'
+down_revision = '9d00917b2fa8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'registration_tags',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('event_id', sa.Integer(), nullable=False, index=True),
```

Comment:
Should they be linked to a particular registration form or the whole event? Not sure tbh.. maybe something to discuss on the channel. I'm fine either way; I guess event-wide has no disadvantages since it's up to the managers when to assign them anyway.